views/layouts/main.html

```diff
@@ -13,6 +13,11 @@
     <div id="refinement-list"></div>
     {{> footer}}
     {{> anchor_links}}
+    <script>
+    if ('serviceWorker' in navigator) {
+      navigator.serviceWorker.register('/sw.js');
```

Comment:
how about naming this file something like `service-worker.js` to make its purpose more obvious?